### PR TITLE
update guides

### PR DESCRIPTION
**Pull request by mitulshah-suse**
Related to https://github.com/rancher/rancher-docs/issues/34

### CODE REVIEW

File: docs/security/self-assessment.md [hunk 1]
```diff
@@ -3,13 +3,13 @@ title: CIS Self Assessment Guide
 weight: 90
 ---
 
-### CIS Kubernetes Benchmark v1.6 - K3s with Kubernetes v1.17 to v1.21
+### CIS Kubernetes Benchmark v1.23 - K3s with Kubernetes v1.22 to v1.24
```
Comment (brandond) [inline docs/security/self-assessment.md:6]: We don't have versioned docs, and still need to support 1.22-1.24, so I don't think we can just wholesale replace the 1.6 benchmark docs with content for 1.23.  Perhaps we should move the old content to `self-assessment-1.6.md` and have a `self-assessment-1.23.md` with the new content?

**Review by brandond — CHANGES_REQUESTED** [review]

**Comment by brandond** [comment]
Don't forget to sign your commits for DCO

File: docs/security/self-assessment.md (hunk 1 above)
Comment (mitulshah-suse) [reply docs/security/self-assessment.md:6]: K3s support is only for versions 1.22-1.24 in the latest versions of rancher 2.6
Older versions 1.17-1.21 are not supported by K3s. So it should be ok to keep only one version for now.
We might need to add versioning later on though.

**Comment by mitulshah-suse** [comment]
> Don't forget to sign your commits for DCO

Done.

**Review by brandond — APPROVED** [review]

**Pull request merged**